zebedee-reader/Dockerfile.concourse

```diff
@@ -6,6 +6,8 @@ ADD dependency target/dependency
 ADD classes target/classes
 
 CMD java -Xmx2048m -cp "target/dependency/*:target/classes/"           \
+    -javaagent:zebedee-cms/target/dependency/aws-opentelemetry-agent-1.30.0.jar \
```

Comment:
Should this be aws-opentelemetry-agent-1.31.0.jar for consistency with the babbage version?